### ISSUE TITLE
cut a v0.3.0 release

### ISSUE THREAD

**Issue by michelleN** [servicemeshinterface/smi-sdk-go#49]
includes the http header matching feature and some bug fixes

**Comment by grampelberg** [comment]
Should we get the API bumps in as well? @stefanprodan any opinions?

**Comment by stefanprodan** [comment]
I would cut a release after #51 is merged.

**Comment by stefanprodan** [comment]
Let's postpone the release after the org move #55 as we need to change the imports.

**Comment by leecalcote** [comment]
Makes sense.

**Comment by stefanprodan** [comment]
0.3.0 has been released
